src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {

```

Comment:
![image](https://github.com/user-attachments/assets/485737ec-be41-4636-a1ee-11c01c8f8f50)

Reply:
Przepraszam nie mam pomysłu, jak to przeprowadzić w ten sposób. Gdy staram się wywołać metodę losującą kolory w metodzie losującej kulę wyrzuca mi błąd. Tak samo z obiektem losowym, który wyciągnąłem poza metodę w klasie ColorSupplier.